code/modules/events/wizard/rpgloot.dm

```diff
@@ -21,7 +23,39 @@
 		else
 			I.name = "[I.name] of [pick(suffixes)]"
 
-		I.force 		+= quality
-		I.force			= max(0,I.force)
-		I.throwforce 	+= quality
-		I.throwforce	= max(0,I.throwforce)
\ No newline at end of file
+		I.force 		= max(0,I.force + quality)
+		I.throwforce	= max(0,I.throwforce + quality)
+		for(var/armor_value in I.armor)
+			armor_value	+= quality
```

Comment:
Ahah! you've fallen into my second trap!

you haven't capped these at 90.
naughty naughty!